### ISSUE TITLE
LKChat block someone for Guild recruitement announce :/

### ISSUE THREAD

**Issue by yotitchy** [Lemon-King/ToS-LKChat#21]
Hi !

1st thx for your addon ! ;)

2nd ! A member of my guild shouted this sentence on Fedimian serv :

> [FR] Rebellion recrute lvl 220+ : Casu ou non, on cherche à avoir un bon gameplay, connaître le jeu et ce qu'il s'y passe. rebellion.ovh !

it's the first time we try this and the poor guy get blocked by everyone
(expect people who got him in friends ? right ?)

so here we are :
- i advice him to make a ticket to TOS to prevent from being ban :/
- i wonder :
  
  > why the addon blocked the announce ?
  > how to pass though ? something like rebellion-dot-ovh ? ovh is taboo ? 
  > is there a list of method of block/filter to avoid ? sort of rules for people like us who just want to announce ? XD
- and then other idea incoming :
  
  > we got an awfull list of blocked ppl : can you add a function to clean the blocked list manually or by period/shedule ?
  > i know you'll say no but i test :p can you, in the next update, if it's easy, add a code to unblock my member ? he is an adept of the shout (worldboss, groups, etc) XD

**Comment by wuaw** [comment]
For me, this phrase is not filtered. I'm using almost the exact same filter as LKChat.

![ss 2016-06-22 at 10 35 46](https://cloud.githubusercontent.com/assets/18574658/16261898/4942585a-3865-11e6-9922-e142953b9ba2.png)

**Comment by yotitchy** [comment]
:/

some people pm-ed him to advertise their addon blocked him, hmm perhaps i miss another addon who do the same think ?

**Comment by wuaw** [comment]
No, I don't think another addon does the same thing. As far as I can see, this text doesn't trigger the filter though.

**Comment by yotitchy** [comment]
![report](https://cloud.githubusercontent.com/assets/13523852/16345171/c665533c-3a40-11e6-95b3-6d83c11db3c6.png)

![report1](https://cloud.githubusercontent.com/assets/13523852/16345207/029dfcbe-3a41-11e6-8e14-c35af1821aaa.jpg)

ok then i get another mumber to get block ( and banned if reported ?) for posting screenshot proof :)

**Comment by wuaw** [comment]
I see, well yeah, I'm not using the exact same filter but very close. My mistake.

**Comment by yotitchy** [comment]
yes, just saw the other addon i dont use :  here its lkchat not classic chat ;)
and looks like in fact that lkchat is now included in classicchat who explain why no answer here ? :D
